tests/fixtures/in-source-config/functions_missing_cron_expression/cron_cjs_invalid_cron_expression.js

```diff
@@ -0,0 +1,5 @@
+const { schedule } = require('@netlify/functions')
+
+module.exports.handler = schedule(null, () => {
+  // function handler
+})
```

Comment:
Can we add one more test that ensures our whole logic works even when another `schedule` is imported and it does not error falsely? By looking at the code this should work already fine:

```js
const { schedule as nfySchedule } = require('@netlify/functions')
const { schedule } = require('other')

schedule();

module.exports.handler = nfySchedule('@daily', async () => {
  // function handler
})
```